generic3g/ESMF_Utilities.F90

```diff
@@ -198,5 +199,22 @@ function to_esmf_state_intent(str_state_intent, rc) result(state_intent)
       _RETURN(_SUCCESS)
    end function to_esmf_state_intent
 
+   function esmf_state_intent_to_string(state_intent, rc) result(state_intent_str)
```

Comment:
Note, I'm fairly certain I've got this function somewhere else in MAPL3.   But let's not let that slow the PR down.